### ISSUE TITLE
Can't install on Raspberry PI

### ISSUE THREAD

**Issue by oalders** [houseabsolute/ubi#41]
Maybe this is a "wontfix", but figured I'd mention it.

```text
curl --silent --location https://raw.githubusercontent.com/houseabsolute/ubi/master/bootstrap/bootstrap-ubi.sh
TARGET=/home/pi/local/bin
sh
boostrap-ubi.sh: Cannot determine what binary to download for your CPU architecture: armv7l
```

```ini
$ cat /etc/os-release
PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"
```

**Comment by autarch** [comment]
Does this build work on the PI? https://github.com/houseabsolute/ubi/releases/download/v0.0.26/ubi-Linux-arm-musl.tar.gz

If it does then this is a very easy fix.

**Comment by oalders** [comment]
Yes, that works. Thanks!

**Comment by autarch** [comment]
I just pushed a change to the bootstrap script that should fix this.

**Comment by oalders** [comment]
Right now I'm getting:

```text
$ curl --silent --location https://raw.githubusercontent.com/houseabsolute/ubi/master/bootstrap/bootstrap-ubi.sh
TARGET=/home/pi/local/bin
sh
downloading https://github.com/houseabsolute/ubi/releases/latest/download/ubi-Linux-arm.tar.gz
curl failed to download https://github.com/houseabsolute/ubi/releases/latest/download/ubi-Linux-arm.tar.gz with status code = 404
```

**Comment by autarch** [comment]
I just pushed another fix.

**Comment by oalders** [comment]
Excellent. Thanks! Fixed via `0.0.26`.
